src/AWS.Deploy.Common/Recipes/Validation/ValidatorFactory.cs

```diff
@@ -68,7 +68,8 @@ public ValidatorFactory(IServiceProvider serviceProvider)
         private static readonly Dictionary<RecipeValidatorList, Type> _recipeValidatorTypeMapping = new()
         {
             { RecipeValidatorList.FargateTaskSizeCpuMemoryLimits, typeof(FargateTaskCpuMemorySizeValidator) },
-            { RecipeValidatorList.ValidDockerfilePath, typeof(DockerfilePathValidator) }
+            { RecipeValidatorList.ValidDockerfilePath, typeof(DockerfilePathValidator) },
+            { RecipeValidatorList.BeanstalkInstanceType, typeof(BeanstalkInstanceTypeValidator) }
```

Comment:
this validator is configured to always run, regardless of what settings the user actually changed right?

Reply:
Yes, Recipe validators always run prior to the start of the deployment. They are not tied to a specific setting. They are usually used to check if multiple settings are compatible.